packages/kbn-optimizer/limits.yml

```diff
@@ -16,7 +16,7 @@ pageLoadAssetSize:
   cloudExperiments: 109746
   cloudFullStory: 18493
   cloudLinks: 55984
-  cloudSecurityPosture: 19109
+  cloudSecurityPosture: 34398
```

Comment:
why this increase in the bundle size? almost 80% percent increase in my opinion signals that smth is off

Reply:
ah, I see, the actual change is +1kb. I am still not sure where it is coming from but it is not a big issue I guess